### ISSUE TITLE
[0.15][Feature] QTable optional columns prop

### ISSUE THREAD

**Issue by nueko** [quasarframework/quasar#1058]
Hi, i think we can make the `columns` prop to be optional that computed from the data, maybe whit `Object.keys(data[0])`

**Comment by rstoenescu** [comment]
Hi, You need a label to display for each column in header. How should this label be determined?

**Comment by nueko** [comment]
In short, we can map it and let all the default options

```js
Object.keys(data[0]).map(field => ({field, label: field..toUpperCase()}))
```

**Comment by rstoenescu** [comment]
Will be available in "quasar" v1.12.0.
